packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -113,7 +113,7 @@ export class Runner {
         let state: ScanState = 'fail';
 
         if (axeResults.results.violations !== undefined && axeResults.results.violations.length > 0) {
-            issueCount = axeResults.results.violations.length;
+            issueCount = axeResults.results.violations.reduce((a, b) => a + b.nodes.length, 0);
```

Comment:
you need to add a proper unit test that validates that you are adding nodes length across violation rules objects